dbt/include/global_project/macros/adapters/common.sql

```diff
@@ -1,11 +1,26 @@
 {% macro adapter_macro(name) -%}
+  {% if '.' in name %}
+    {% set package_name, name = name.split(".", 1) %}
+  {% else %}
+    {% set package_name = none %}
+  {% endif %}
+
+  {% if package_name is none %}
+    {% set package_context = context %}
+  {% elif package_name in context %}
+    {% set package_context = context[package_name] %}
+  {% else %}
+    {{ exceptions.raise_compiler_error("Bad context for adapter macro: " ~ package_name) }}
```

Comment:
Am I right in thinking that the usage of this would be:

```
{{ adapter_macro('package.macro', *args) }}
```

If so, this error message is not all that useful. How about "In adapter_macro: could not find package '{package_name}', called with '{original_name}'"

Reply:
@cmcarthur good idea. Just fixed:

```
$ dbt run --model test_model
Encountered an error:
Compilation Error
  In adapter_macro: could not find package 'my_other_project', called with 'my_other_project.current_timestamp'
```